app/jvm/src/main/scala/app/models/quiz/config/QuizConfigParsableValue.scala

```diff
@@ -449,6 +449,7 @@ class QuizConfigParsableValue @Inject() (implicit
       "question" -> Required(StringValue),
       "questionDetail" -> Optional(StringValue),
       "masterNotes" -> Optional(StringValue),
+      "image" -> Optional(ImageValue),
```

Comment:
This will break this test: app/jvm/src/test/scala/app/models/quiz/config/QuizConfigParsableValueTest.scala